Xamarin.Forms.Core/Xamarin.Forms.Core.csproj

```diff
@@ -21,7 +21,7 @@
 	</ItemGroup>
 	<Import Project="..\Xamarin.Flex\Xamarin.Flex.projitems" Label="Shared" Condition="Exists('..\Xamarin.Flex\Xamarin.Flex.projitems')" />
 	<UsingTask TaskName="XFCorePostProcessor.Tasks.FixXFCoreAssembly" AssemblyFile="..\XFCorePostProcessor.Tasks\bin\Debug\netstandard2.0\XFCorePostProcessor.Tasks.dll" />
-	<Target AfterTargets="AfterCompile" Name="ShameHat">
+	<Target  Condition=" ! ($(DesignTimeBuild) == true OR $(BuildingProject) != true)" AfterTargets="AfterCompile" Name="ShameHat">
```

Comment:
that isolated `!` look out of place, couldn't you change it to `!= true` ?

Reply:
Yea basically I just took what they had here
https://github.com/dotnet/project-system/blob/master/docs/design-time-builds.md#determining-whether-a-target-is-run-in-a-design-time-build

And then added the *!* to the whole thing. Mainly because I've messed up trying to simplify these msbuild conditions down in the past since they can also be ('') so just went the easiest route

That being said I still just checked in a simplified version that should be the same